make/autoconf/lib-bundled.m4

```diff
@@ -62,19 +62,23 @@ AC_DEFUN_ONCE([LIB_SETUP_LIBJPEG],
 
   if test "x${with_libjpeg}" = "xbundled"; then
     USE_EXTERNAL_LIBJPEG=false
+    LIBJPEG_CFLAGS=""
+    LIBJPEG_LIBS=""
   elif test "x${with_libjpeg}" = "xsystem"; then
-    AC_CHECK_HEADER(jpeglib.h, [],
```

Comment:
It's okay that you add `PKG_CHECK_MODULES`, but please keep the `AC_CHECK_HEADER` and `AC_CHECK_LIB` to verify that it actually works.